types/testingutils/comparable/instance.go

```diff
@@ -20,28 +17,46 @@ func SetMessages(instance *qbft.Instance, messages []*types.SSVMessage) {
 		if err := msg.Decode(ssvMsg.Data); err != nil {
 			panic(err.Error())
 		}
+		setMessage(instance, msg)
+	}
+}
+
+func SetSignedMessages(instance *qbft.Instance, messages []*qbft.SignedMessage) {
+	initContainers(instance)
```

Comment:
this is new, is it used anywhere?

Reply:
See https://github.com/bloxapp/ssv-spec/pull/210#discussion_r1196025399